### DISCUSSION TITLE
allow swiping left and right if modal height is beyond viewport / scrollable?

### DISCUSSION THREAD

**Discussion by steverethink** [fancyapps/ui#102] [Q&A]
i seem to recall this was an issue in previous fancybox's but was rectified at some point.

i'm currently putting an 'overflow:auto' on the .fancy__slide class so the modal window just expands past the viewport height and allow it to be scrollable up and down. 

this seems to disable swiping left and right on mobile though. 

i know both actions shouldn't really be mixed, but is there a quick way around this limitation?

**Comment by fancyapps** [comment]
Hi,

Sorry about the late replay. I've been thinking about it and I couldn't quickly make it work well. I've also thought about rewriting the Panzoom component to make it work for scrollable content. I will keep in mind your feature request, but I can not promise any ETA.
